core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -137,11 +137,23 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
             diplomacyTable.add(nextLevelString.toLabel()).row()
         }
 
+        val cultureBonus = if (viewingCiv.getEraNumber() in 0..1) 3f else if (viewingCiv.getEraNumber() in 2..3) 6f else 13f
+        val happinessBonus = if (viewingCiv.getEraNumber() in 0..1) "2" else "3"
```

Comment:
I really don't like that the amount is based on the number of the era.
This should probably be a unique for each era.